bin/compile

```diff
@@ -134,6 +134,41 @@ else
   CACHED_PYTHON_STACK=$STACK
 fi
 
+# TODO: Move this into a new package manager handling implementation when adding Poetry support.
+if [[ ! -f requirements.txt && ! -f Pipfile && ! -f setup.py ]]; then
+  puts-warn
+  puts-warn "Error: No supported Python package manager files were found."
+  puts-warn
+  # We intentionally don't mention `setup.py` here since it's being removed soon.
```

Comment:
How soon is soon? If we're talking longer than a day or two, maybe you should mention it until it truly is removed?

Reply:
I'll be removing it in the next week. However, I don't want new projects to start using it, so regardless of timeline I don't want to advertise it. (Only <0.1% of builds use it currently.)